### PR TITLE
add build instructions with LLVM-11

### PR DESCRIPTION
**Pull request by 251**
Installation instructions were tested in a Ubuntu Impish Docker container with:
* https://github.com/klee/klee-uclibc/pull/41, and
* https://github.com/klee/klee/pull/1493

### CODE REVIEW

**Comment by ccadar** [comment]
@251 instead of deleting `build-llvm9.md` and adding `build-llvm11.md`, can you instead do a rename so that we can see the changes?

**Comment by 251** [comment]
I already used `git mv` - is there another way?

**Comment by ccadar** [comment]
Somehow I recall that previous renames were properly handled, but your link suggests that you cannot really rename.  Perhaps what happened before is that the rename and the change were in different commits, so the changes were clearly displayed in the GitHub interface.

**Comment by 251** [comment]
The second commit shows the changes (ignore whitespace, I made the formatting more consistent).

**Comment by ccadar** [comment]
Ah, interesting, good to know that the commit view does this.

**Comment by ccadar** [comment]
But I see you split it now into two commits, there was a single commit before.  Now it makes sense, it's consistent with what I mentioned before.

**Comment by 251** [comment]
Yes, that's why I split the commits. Not as nice but works.

File: build-llvm11.md [hunk 1]
```diff
@@ -181,13 +171,14 @@ POSIX environment under macOS.  KLEE does not work under x86-32.
    Here's an example:
 
    ```
-/usr/lib/llvm-3.4/include/llvm/Support/CommandLine.h:905: undefined reference to `vtable for llvm::cl::parser<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >'
-CMakeFiles/kleaver.dir/main.cpp.o: In function `main':
-/home/user/programs/klee/klee/tools/kleaver/main.cpp:413: undefined reference to `llvm::error_code::message[abi:cxx11]() const'
-CMakeFiles/kleaver.dir/main.cpp.o: In function `llvm::cl::opt<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, false, llvm::cl::parser<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > >::~opt()':
+   /usr/lib/llvm-3.4/include/llvm/Support/CommandLine.h:905: undefined reference to `vtable for llvm::cl::parser<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >'
```
Comment (ccadar) [inline build-llvm11.md:174]: I wonder if this is still relevant.  The example shows LLVM 3.4

Comment (251) [reply build-llvm11.md:174]: Me as well... shouldn't happen on current systems imho.

Comment (251) [reply build-llvm11.md]: added a commit that removes this section

File: build-llvm11.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+   ```
+
+   You should also install `lit` to enable testing, `tabulate` to support additional features in `klee-stats` and `wllvm` to make it easier to compile programs to LLVM bitcode:
+
+   ```bash
+   $ sudo pip3 install lit wllvm
+   $ sudo apt-get install python3-tabulate
+   ```
+
+   Use `--user` for `pip3` to install packages for the current user only and make sure that `~/.local/bin` is in your `PATH`.
```
Comment (MartinNowack) [inline build-llvm11.md:36]: `--user` without `sudo`

And the proper check is:
```
base_path=$(python3 -m site --user-base)
export PATH="${base_path}/bin:$PATH"
```
As it's different under macOS

File: build-llvm11.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+
+9. **Build KLEE:**
+
+   From the ``build`` directory created in the previous step run:
+
+   ```bash
+   $ make
+   ```
+
+   <a name="rtti_link_error">**NOTE:**</a> If you see linker errors involving undefined references to `typeinfo` this is likely an [RTTI issue](https://github.com/klee/klee/issues/508).
```
Comment (MartinNowack) [inline build-llvm11.md:170]: This part of the section should not be needed anymore.
The reporting should be accurate by LLVM 6

**Comment by MartinNowack** [comment]
@251 Can you fix the missing things?

**Comment by 251** [comment]
@MartinNowack Sure, let me just check if it still works.

**Pull request merged**